_extras/extra_challenges.md

```diff
@@ -0,0 +1,38 @@
+---   
+layout: page                                                                                            
+title: "Extra Challenges"                                                                           
+permalink: /extra_challenges/                                                                            
+---                                                                                                     
+
+# Extra Challenges
+
+A collection of challenges that have been either removed from or not (yet) added to the main lesson. 
+
+> ## Looping Over Dataframe
+>
+> (Please refer to lesson `06-loops-and-functions.md`)
+>
+> The file `surveys.csv` in the `data` folder contains 25 years of data from surveys,
+> starting from 1977. We can load the data and print all the years surveyed using a `for` loop:
+> 
+> ~~~
+> import pandas as pd
+> 
+> # Load the data into a DataFrame
+> surveys_df = pd.read_csv('data/surveys.csv')
+> 
+> # Loop through a sequence of years and print the year
+> start_year = 1977
+> end_year = 2002
+> for year in range(start_year, end_year+1):
+>     print(year)
+> ~~~
+> {: .language-python}
+>
+> What happens if there is no data for a year in a sequence? For example,
+> imagine we used `1976` as the start year in `range`
```

Comment:
```suggestion
> imagine we used `1976` as the year in `surveys_df[surveys_df.year == year]`
```

I think you're exactly correct on what the challenge _meant_ to do. As it's written ... nothing special happens. You just start the `range` function one year earlier. I've made a suggestion here that makes the answer make sense. 

Sorry to make you do the labor on this as someone who just noticed the challenge was a little goofy;)

Reply:
Nice call @wrightaprilm, thanks a lot!